usage.go

```diff
@@ -78,26 +78,37 @@ func (p *Parser) WriteHelp(w io.Writer) {
 	}
 
 	// write the list of options
-	if len(options) > 0 {
-		fmt.Fprint(w, "\noptions:\n")
-		const colWidth = 25
-		for _, spec := range options {
-			left := "  " + synopsis(spec, "--"+spec.long)
-			if spec.short != "" {
-				left += ", " + synopsis(spec, "-"+spec.short)
-			}
-			fmt.Fprint(w, left)
-			if spec.help != "" {
-				if len(left)+2 < colWidth {
-					fmt.Fprint(w, strings.Repeat(" ", colWidth-len(left)))
-				} else {
-					fmt.Fprint(w, "\n"+strings.Repeat(" ", colWidth))
-				}
-				fmt.Fprint(w, spec.help)
-			}
-			fmt.Fprint(w, "\n")
+	fmt.Fprint(w, "\noptions:\n")
+	for _, spec := range options {
+		printOption(w, spec)
+	}
+
+	// write the list of built in options
+	dest := reflect.ValueOf(true)
+	builtins := []*spec{
```

Comment:
rather than making a list of builtins and then iterating over the, how about just

```go
printOption(w, &spec{long: "help", short: "h", help: "display this help and exit"})
```

If we add more builtins, I think it's okay to simply duplicate this line for each one.